stdlib/html/parser.pyi

```diff
@@ -28,3 +29,18 @@ class HTMLParser(ParserBase):
     def parse_pi(self, i: int) -> int: ...  # undocumented
     def parse_starttag(self, i: int) -> int: ...  # undocumented
     def set_cdata_mode(self, elem: str) -> None: ...  # undocumented
+    def parse_comment(self, i: int) -> int: ...  # undocumented
+    def parse_declaration(self, i: int) -> int: ...  # undocumented
+    def parse_marked_section(self, i: int) -> int: ...  # undocumented
+    def updatepos(self, i: int, j: int) -> int: ...  # undocumented
+    rawdata: str  # undocumented
+    cdata_elem: str | None  # undocumented
+    convert_charrefs: bool  # undocumented
+    interesting: Pattern[str]  # undocumented
+    lasttag: str  # undocumented
+    lineno: int  # undocumented
+    offset: int  # undocumented
+    if sys.version_info >= (3, 10): ...
+    else:
+        # Removed from ParserBase: https://bugs.python.org/issue31844
+        def error(self, message: str) -> None: ...  # undocumented
```

Comment:
Feels a bit odd not writing this as:
```python
if sys.version_info < (3, 10):
    def error(self, message: str) -> None: ... # undocumented
```
but I've had issues with mypy struggling `<` rather than `>=` in a separate project recently ([here](https://github.com/sqlfluff/sqlfluff/pull/2147#issuecomment-998329796)) so figured it's best to write use the format in the PR to be safe.

Happy to follow your suggestion here.

Reply:
We use the `if sys.version_info < (3, 10):` construct in typeshed quite a bit, so it should be fine.

Reply:
thanks, I've switched back to the simple format

Reply:
I don't think mypy supports `(3, 8, 0)`, or at least we don't use the longer versions in typeshed. Changing that to `(3, 8)` is likely what fixed the problem in sqlfluff.

Reply:
ahh nice thanks for clarifying! One of those weird edge cases that make you second guess something basic 😄